.circleci/config.yml

```diff
@@ -83,23 +80,17 @@ version: 2
 jobs:
   # Build Arnold's docker image
   build:
-    # We use the machine executor, i.e. a VM, not a container
-    machine:
-      # Prevent cache-related issues
-      docker_layer_caching: false
-
-    working_directory: ~/fun
+    <<: *defaults
 
     steps:
       - checkout
-      - *ci_env
       - *docker_login
       - run:
           name: Build arnold production image
```

Comment:
The name is not accurate anymore with what you are doing

Reply:
Do you suggest making it plural?

Reply:
This is more accurante